src/sentry/snuba/events.py

```diff
@@ -15,7 +15,7 @@ class Columns(Enum):
 
     EVENT_ID = Column("events.event_id", "event_id", "event_id", "event_id", "id")
     GROUP_ID = Column("events.group_id", "group_id", None, "group_id", "issue.id")
-    ISSUE = Column("events.issue", "issue", None, "group_id", "issue.id")
+    # ISSUE = Column("events.issue", "issue", None, "group_id", "issue.id")
```

Comment:
This field could be saved in discover1 queries. Should we replace its events alias with `group_id` instead?

Reply:
Don't we just save the public names in Discover queries? Since the public name is the same as group_id in the line above, we should be good?

Reply:
You're right I missed that the group_id column had the same public alias.